resources/views/models/custom_fields_form_bulk_edit.blade.php

```diff
@@ -0,0 +1,113 @@
+                              {{ $value }}
+                          </label>
+                      </div>
+                  @endforeach
+            @elseif ($field->element=='radio')
+            @foreach ($field->formatFieldValuesAsArray() as $value)
+              <div>
+                  <label>
+                      <input type="radio" value="{{ $value }}" name="{{ $field->db_column_name() }}" class="minimal" {{ isset($item) ? ($item->{$field->db_column_name()} == $value ? ' checked="checked"' : '') : (Request::old($field->db_column_name()) != '' ? ' checked="checked"' : (in_array($value, explode(', ', $field->defaultValue($model->id))) ? ' checked="checked"' : '')) }}>
+                      {{ $value }}
+                  </label>
+              </div>
+            @endforeach
+
+            @endif
+
+            @else
+            <!-- Date field -->
+
+            @if ($field->format=='DATE')
+
+            <div class="input-group col-md-5" style="padding-left: 0px;">
+                <div class="input-group date" data-provide="datepicker" data-date-format="yyyy-mm-dd" data-autoclose="true" data-date-clear-btn="true">
+                    <input type="text" class="form-control" placeholder="{{ trans('general.select_date') }}" name="{{ $field->db_column_name() }}" id="{{ $field->db_column_name() }}" readonly value="{{ old($field->db_column_name(),(isset($item) ? Helper::gracefulDecrypt($field, $item->{$field->db_column_name()}) : $field->defaultValue($model->id))) }}"  style="background-color:inherit">
+                    <span class="input-group-addon"><i class="fas fa-calendar" aria-hidden="true"></i></span>
+                </div>
+            </div>
+
+
+                @else
+                    @if (($field->field_encrypted=='0') || (Gate::allows('admin')))
+                    <input type="text" value="{{ Request::old($field->db_column_name(),(isset($item) ? Helper::gracefulDecrypt($field, $item->{$field->db_column_name()}) : $field->defaultValue($model->id))) }}" id="{{ $field->db_column_name() }}" class="form-control" name="{{ $field->db_column_name() }}" placeholder="Enter {{ strtolower($field->format) }} text">
+                        @else
+                            <input type="text" value="{{ strtoupper(trans('admin/custom_fields/general.encrypted')) }}" class="form-control disabled" disabled>
+                    @endif
+                @endif
+
+          @endif
+
+        @if ($field->help_text!='')
+            <p class="help-block">{{ $field->help_text }}</p>
+        @endif
+
+
+        <p>On Models:
+             @foreach($field->assetModels() as $assetModel) 
+                @if(in_array($assetModel->name, $modelNames)) 
+                    {{$assetModel->name}}{{($loop->last) ? '' : ', '}}
+                @endif 
+            @endforeach 
```

Comment:
I had some extra `,`s here:
![CleanShot 2023-06-27 at 11 00 31@2x](https://github.com/snipe/snipe-it/assets/1141514/08f15292-de87-47ed-91b0-034511536d6e)

Since `$field->assetModels()` returns a collection something like this should work:
```php
$assetModels->pluck('name')->intersect($modelNames)->implode(', ');
```

Reply:
So I had the same problem, but it went away - so I assume it's a caching issue or something
<img width="653" alt="image" src="https://github.com/snipe/snipe-it/assets/7305753/f98acb09-5bc8-4ba3-889b-e9d5d87fc62d">

Reply:
Ah....`{{($loop->last) ? '' : ', '}}` is throwing commas is actually doing the inverse of what we expect.

If you throw the following snippet inside of `@foreach($field->assetModels() as $assetModel)`

```
<table class="table">
    <tr>
        <th>Iteration</th>
        <th>Remaining</th>
        <th>Comma?</th>
        <th>Model Name</th>
        <th>In $modelNames</th>
    </tr>
    <tr>
        <td>{{ $loop->iteration }}</td>
        <td>{{ $loop->remaining }}</td>
        <td>{{($loop->last) ? '' : ', '}}</td>
        <td>{{ $assetModel->name }}</td>
        <td>@if(in_array($assetModel->name, $modelNames)) true @else false @endif</td>
    </tr>
</table>
```

you'll see the following:
![CleanShot 2023-06-28 at 10 54 02@2x](https://github.com/snipe/snipe-it/assets/1141514/138616fd-04b2-4b4e-9939-ec69d7f94d5b)